OPHD/UI/PopulationPanel.h

```diff
@@ -4,6 +4,8 @@
 
 #include <NAS2D/Renderer/RectangleSkin.h>
 
+#include <libOPHD/Population/PopulationPool.h>
```

Comment:
Consider using a forward declare for `PopulationPool` rather than a `#include`. Similar to `Population`, we don't need a complete type in this file. Using a forward declare here, and moving the `#include` to the `.cpp` will decrease transitive includes, which helps lower compile times.